Tests/MetricsTests/TestMetrics.swift

```diff
@@ -176,12 +182,21 @@ internal class TestTimer: TimerHandler, Equatable {
     }
 }
 
-private extension NSLock {
-    func withLock<T>(_ body: () -> T) -> T {
+extension NSLock {
+    fileprivate func withLock<T>(_ body: () -> T) -> T {
         self.lock()
         defer {
             self.unlock()
         }
         return body()
     }
 }
+
+// MARK: - Sendable support
+
+#if compiler(>=5.6)
+// ideally we would not be using @unchecked here, but concurrency-safety checks do not recognize locks
```

Comment:
it was because touching static things right? because just touching class state even warn concurrency should be just ignoring if in a class I think 🤔

Reply:
no it does warn about touching state of class